sourcemash/categorize.py

```diff
@@ -20,4 +22,9 @@ def categorizeItem(title, text, category_dict):
 	# Return top 2 words in categories
 	most_frequent = categories.most_common(2)
 
-	return (most_frequent[0][0], most_frequent[1][0])
\ No newline at end of file
+	if len(most_frequent) > 0:
```

Comment:
Can you write a test for this part? Just so we don't change it in the future with unexpected consequences.

i.e. set text to "the of" and make sure "" and "" are returned.